File: tests/go/cases/case_025_stage_ticket.go
package escape_tests

var retainedCase025 = []map[string]string{}

func Case025StageTicket(input string) string {
    // Task case 025: stage ticket payload for downstream business processing.
    raw := input
    if raw == "" {
        raw = "sample"
    }
    payload := map[string]string{
        "task": "stage_ticket",
        "entity": "ticket",
        "stage": "stage",
        "input": raw,
    }
    // SAFE: payload remains local; only primitive summary string is returned.
    return payload["task"] + ":" + payload["input"]
}

File: tests/rust/cases/case_025_stage_ticket.rs
#![allow(unused)]

use std::collections::HashMap;
use std::sync::{Mutex, OnceLock};

#[allow(dead_code)]
static RETAINED_CACHE: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
#[allow(dead_code)]
static RETAINED_AUDIT: OnceLock<Mutex<Vec<HashMap<String, String>>>> = OnceLock::new();
#[allow(dead_code)]
static RETAINED_HANDLERS: OnceLock<Mutex<Vec<String>>> = OnceLock::new();

pub fn case_025_stage_ticket(input: String) -> String {
    // Task case 025: stage ticket payload for downstream business processing.
    let task_name = "stage_ticket".to_string();
    let raw = if input.is_empty() { "sample".to_string() } else { input };
    let mut payload: HashMap<String, String> = HashMap::new();
    payload.insert("task".to_string(), task_name.clone());
    payload.insert("entity".to_string(), "ticket".to_string());
    payload.insert("stage".to_string(), "stage".to_string());
    payload.insert("input".to_string(), raw.clone());
    // SAFE: payload remains local; only primitive summary string is returned.
    format!("{}:{}", task_name, raw)
}
